{"name": "Class PrinterIOException", "module": "java.desktop", "package": "java.awt.print", "text": "The PrinterIOException class is a subclass of\n PrinterException and is used to indicate that an IO error\n of some sort has occurred while printing.\n\n As of release 1.4, this exception has been retrofitted to conform to\n the general purpose exception-chaining mechanism.  The\n \"IOException that terminated the print job\"\n that is provided at construction time and accessed via the\n getIOException() method is now known as the cause,\n and may be accessed via the Throwable.getCause() method,\n as well as the aforementioned \"legacy method.\"", "codes": ["public class PrinterIOException\nextends PrinterException"], "fields": [], "methods": [{"method_name": "getIOException", "method_sig": "public IOException getIOException()", "description": "Returns the IOException that terminated\n the print job.\n\n This method predates the general-purpose exception chaining facility.\n The Throwable.getCause() method is now the preferred means of\n obtaining this information."}, {"method_name": "getCause", "method_sig": "public Throwable getCause()", "description": "Returns the cause of this exception (the IOException\n that terminated the print job)."}]}